wrangler.toml

```diff
@@ -2,3 +2,4 @@ name = "discord-slash-commands"
 main = "src/index.ts"
 compatibility_date = "2023-01-13"
 node_compat = true
+[vars]
```

Comment:
could you point me to a document and explain why `[vars]` is required

Reply:
That popped up when I updated the environment variable on cloudflare. 
I tried testing the function without `[vars]` and it is working without any glitch, so I have removed `[vars]` from `wrangler.toml` file 

Thanks